Content.Server/Ensnaring/EnsnareableSystem.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.Server.Ensnaring.Components;
+using Content.Server.Popups;
+using Content.Shared.Ensnaring;
+using Content.Shared.Ensnaring.Components;
+using Content.Shared.Popups;
+using Robust.Server.Containers;
+using Robust.Shared.Containers;
+using Robust.Shared.Player;
+
+namespace Content.Server.Ensnaring;
+
+public sealed class EnsnareableSystem : SharedEnsnareableSystem
+{
+    [Dependency] private readonly ContainerSystem _container = default!;
+    [Dependency] private readonly PopupSystem _popup = default!;
+    [Dependency] private readonly EnsnaringSystem _ensnaring = default!;
```

Comment:
Just make the system a partial.

Reply:
I can't make it a partial, it won't work that way.